taskcluster/ci/build/kind.yml

```diff
@@ -51,6 +54,13 @@ job-defaults:
                     - ':{component}:lint'
                     - ':{component}:publish'
                     - ':{component}:renameSnapshotArtifacts'
+                nightly:
+                    - '-PnightlyVersion={nightlyVersion}'
+                    - ':{component}:assemble'
```

Comment:
I assume you just copied that from `snapshot` - but I do wonder how we ended up using `assemble`. I assume this will build debug and release version, even though, I think, we will only need releases (meaning `assembleRelease`)?

Reply:
@pocmo  Yeah, I actually wasn't sure what the difference was so I went witht the `snapshot` functionality (copy-paste). Should I change `assemle` -> `assembleRelease` and then the same for `testRelease`, `lintRelease`?

Reply:
Yeah, I think that would make sense and save us so time. Let's try that :)

Reply:
Done